Zend/tests/assert/disable_assert_function.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Use disable_functions INI setting to disable assert()
```

Comment:
This test seems unrelated, do we still need it? If so, would it make sense to merge it separately?